charts/keycloak/values.yaml

```
@@ -166,6 +166,16 @@ readinessProbe: |
   initialDelaySeconds: 30
   timeoutSeconds: 1
 
+# Startup probe configuration
+startupProbe: |
+  httpGet:
+    path: /auth/
+    port: http
+  initialDelaySeconds: 30
+  timeoutSeconds: 1
+  failureThreshold: 30
```

```suggestion
  failureThreshold: 60
```
This should be 60 to keep the 300 second timeout from before.